introduction.md

```diff
@@ -0,0 +1 @@
+Hey, this is a Code School project. Nice to meet you! My favorite code school path is Ruby.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.